sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/TextAnalyticsAsyncClient.java

```diff
@@ -512,7 +516,6 @@ public TextAnalyticsServiceVersion getServiceVersion() {
      */
     @ServiceMethod(returns = ReturnType.COLLECTION)
     public TextAnalyticsPagedFlux<LinkedEntity> recognizeLinkedEntities(String document, String language) {
-        Objects.requireNonNull(document, "'document' cannot be null.");
```

Comment:
It appears you've removed the check for null documents and haven't added in a call to your inputDocumentsValidation utility function (also seeing other instances of this)

Reply:
Async API will never throw an exception but return the error. It is handled in https://github.com/Azure/azure-sdk-for-java/pull/10593/files/35656e916f51de922508097123cab21513543620#diff-b7e39c6a437cc9256cc5cf809bf8531dR855